README.md

```diff
@@ -0,0 +1,227 @@
+# Ansible Role: Receptor
+
+Installs and configures a Receptor node on RHEL/CentOS/Fedora servers.
+
+
+## Role Variables
+
+Available variables are listed below, along with default values.
+
+---
+
+    receptor_packages:
+      - receptor
+      - receptorctl
+
+Set the names of the packages needed to install Receptor.
+
+---
+
+    receptor_dependencies: []
+
+Specify other packages needed, probably on a per-node-type basis using
+groupvars or hostvars.
+
+---
+
+    receptor_user: 'receptor'
+    receptor_group: 'receptor'
+
+The user and group under which Receptor will run.
+
+---
+
+    receptor_socket_dir: '/var/run/receptor'
+
+The directory that Receptor will place its control socket into.
+
+---
+
+    receptor_control_filename: 'receptor.sock'
+
+The name of the control socket file.
+
+---
+
+    receptor_config_path: '/etc/receptor'
+
+Path to the Receptor config file.
+
+---
+
+    receptor_control_service: 'control'
```

Comment:
this variable should not be allowed to be configured, it needs to be 'control' in order for receptor to work properly

we can change it here, but also need to change it in installer case

Reply:
Why does Receptor even allow the name to change then?  Should we rip that part of Receptor's configuration language out?